### ISSUE TITLE
Send POST command - results to Status 415 - "Unsupported Media Type"

### ISSUE THREAD

**Issue by mj1241** [OpenWonderLabs/SwitchBotAPI#90]
Hi,

Background/Description:
I would like to ask for support. I think it may not be a bug since I could not find any similar issue and my issue is pretty basic to be a bug, but I am having hard time to figure out what is wrong.
I was able to use the list device and get status request, but my problem currently is upon trying POST command request, I am getting Unsupported Media Type as error status. I checked the request path and the request header compared to documentation but it doesn't seem to be wrong. 

**To Reproduce:**
1. I use "Postman" Application to send web API request
2. Select POST as command type, then input the Post Command path with my target device parameter I took from the list I got from my GET device list query 
![image](https://user-images.githubusercontent.com/102502901/160331521-eb753aea-71e9-4ccd-b451-9e7b0371cb67.png)

3. I put on the header the authorization code I got from my phone, it is same a the code i used when i executed GET device and GET status
![image](https://user-images.githubusercontent.com/102502901/160331725-4fef3f73-d2cd-4e7b-8b76-acc8e125b20c.png)
 
4. I input command details on the body:
![image](https://user-images.githubusercontent.com/102502901/160331874-88c46c16-c620-41ab-9ca2-4049cdfc7216.png)
* I also tried some combination like omit the optional, make optional parameter value blank, but same output I got.

5. click send Button

**Expected behavior:**
I am expecting that I can get a success response but I got Unsupported Media Type instead

**Screenshots:**
![botCommandHeader](https://user-images.githubusercontent.com/102502901/160332305-4b54dcff-9b61-4034-a128-4f3c0ff1469b.PNG)
![botCommandBody](https://user-images.githubusercontent.com/102502901/160332327-f3f38746-c52f-4971-a1be-d293a8c20bc4.PNG)
reference screenshot for status and device availability:
![botDevices](https://user-images.githubusercontent.com/102502901/160332370-ca6eefbf-5fdc-41e8-a234-77d48aab7fe9.PNG)
![botStatus](https://user-images.githubusercontent.com/102502901/160332388-6460694d-cdc7-43fc-9288-0b71d40c836b.PNG)

**Device Model:**
SwitchBot Hub Mini (W0202200) and SWITCHBOT-S1

# **SwitchBotAPI**

- **Use Case**:
Send Simple On/Off Command

**Comment by ganlbarone** [comment]
try adding content-type:application/json to the header.

**Comment by mj1241** [comment]
Hi,

Thank you for your reply.
I reviewed also the documentation and found out that I missed out the note for 
POST, it needs content type.
![image](https://user-images.githubusercontent.com/102502901/160503846-c21738d2-9571-41ec-aedd-ce9e396f0981.png)

Then using Postman Application, I also found out that I need to change the format of body from x-www-form-urlencoded to raw.  Write the body parameter in JSON format with "" on the key and value
![image](https://user-images.githubusercontent.com/102502901/160504159-3efbc644-be03-4755-a434-30d48de00995.png)
 
I added the Content-Type to the header as documentation said:
![image](https://user-images.githubusercontent.com/102502901/160504247-1491dc47-441f-47dc-bff3-1218659efa7c.png)

and it can no controls the switch
